<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QMainWindow" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车载多媒体-视频</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#btn_mode[mode='pause']{
	border-image: url(:/resource/video/play.png);
}
#btn_mode[mode='play']{
	border-image: url(:/resource/video/pause.png);
}
#playbackSpeed {
        padding-right: 5px; /* 可选：避免文字贴边 */
        border: none;
        background-color: transparent;
    }

#playbackSpeed :down-arrow {
        image: none; /* 隐藏默认箭头 */
    }

#playbackSpeed :drop-down {
        border: none;
        background: transparent;
    }
#playbackSpeed {
    border: 2px solid rgb(0, 0, 0); /* 边框宽度、类型、颜色 */
    padding: 2px;              /* 可选：内边距，让内容不贴边 */
    border-radius: 4px;        /* 可选：圆角边框 */
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="btn_pre">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/pre.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>540</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/next.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>545</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>520</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="local">
     <attribute name="title">
      <string>本地列表</string>
     </attribute>
     <widget class="QListWidget" name="localList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>220</width>
        <height>490</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="online">
     <attribute name="title">
      <string>在线列表</string>
     </attribute>
     <widget class="QListWidget" name="onlineList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>220</width>
        <height>490</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_loadFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/load.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="voiceSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>130</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fullScreen">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>545</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/full.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="playbackSpeed">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_voiceMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>555</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/voiceMin.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_voiceMax">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>555</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/video/voiceMax.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
